client/src/actions/newGrant.ts

```diff
@@ -20,7 +20,19 @@ export interface GrantCreated {
   owner?: string;
 }
 
-export type NewGrantActions = GrantCreated | NewGrantTXStatus;
+export const RESET_TX_STATUS = "RESET_TX_STATUS";
+export interface IPFSResetTXStatus {
+  type: typeof RESET_TX_STATUS;
```

Comment:
@schultztimothy  I would rename `RESET_TX_STATUS` to have the prefix like all the other actions